### PR TITLE
Add mai/matic deprecation proposal.

### PR DESCRIPTION
**Pull request by reem**
The avalanche portion still fails testing with the `CALLER_NOT_RISK_OR_POOL_ADMIN` error, presumably because the executor should be the guardian on avalanche? Let us know what the right fix here would be in the new system.

### CODE REVIEW

File: src/20231031_Multi_MAIMIMATICDeprecation20231031/AaveV3Optimism_MAIMIMATICDeprecation20231031_20231031.sol [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    override
+    returns (IAaveV3ConfigEngine.CollateralUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.CollateralUpdate[]
+      memory collateralUpdate = new IAaveV3ConfigEngine.CollateralUpdate[](1);
+
+    collateralUpdate[0] = IAaveV3ConfigEngine.CollateralUpdate({
+      asset: AaveV3OptimismAssets.MAI_UNDERLYING,
+      ltv: EngineFlags.KEEP_CURRENT,
+      liqThreshold: 65_00,
```
Comment (rustboyar) [inline src/20231031_Multi_MAIMIMATICDeprecation20231031/AaveV3Optimism_MAIMIMATICDeprecation20231031_20231031.sol:55]: forum post differs from the snapshot and says LT will be 0.3
probably makes sense to mention the change on the forum

**Comment by rustboyar** [comment]
please merge main, do forge install and re-run tests for avalanche
+ add snapshot links

**Comment by reem** [comment]
Rebased on main, updated config and added the snapshot url, and regenerated. This fixed the issue with avalanche but now the optimism payload fails during the default test when trying to run the e2eTest at the end of the proposal. Appears to be error `59` or `PRICE_ORACLE_SENTINEL_CHECK_FAILED` - not sure exactly what's going on here, could use your help understanding what to do next.

**Comment by Zer0dot** [comment]
> Rebased on main, updated config and added the snapshot url, and regenerated. This fixed the issue with avalanche but now the optimism payload fails during the default test when trying to run the e2eTest at the end of the proposal. Appears to be error `59` or `PRICE_ORACLE_SENTINEL_CHECK_FAILED` - not sure exactly what's going on here, could use your help understanding what to do next.

Hey try updating the fork block to 111887045, it seems the network may have been down for the selected block.

File: src/20231031_Multi_MAIMIMATICDeprecation20231031/AaveV3Optimism_MAIMIMATICDeprecation20231031_20231031.sol (hunk 1 above)
Comment (Zer0dot) [reply src/20231031_Multi_MAIMIMATICDeprecation20231031/AaveV3Optimism_MAIMIMATICDeprecation20231031_20231031.sol:55]: @reem

Comment (reem) [reply src/20231031_Multi_MAIMIMATICDeprecation20231031/AaveV3Optimism_MAIMIMATICDeprecation20231031_20231031.sol:55]: Thanks, we've updated the forum post to match

**Comment by reem** [comment]
Updated blocks and re-ran tests, that worked thanks!

**Review by Zer0dot — DISMISSED** [review]

**Review by rustboyar — DISMISSED** [review]

**Comment by reem** [comment]
Added deployed addresses for all chains.

To confirm, with gov 2.5 and beyond avalanche payloads are now executed by normal governance and not by a steward?

**Comment by reem** [comment]
Updated blocks and included requested parameter changes, as detailed in the updated proposal description.

**Comment by reem** [comment]
I tried to merge main and update for changes to the generator, but I got some errors when trying to regenerate with this config, is there a summary of changes to the config that I should apply here?

**Comment by reem** [comment]
The payloads will also need to be re-deployed after further review

**Comment by reem** [comment]
Added newly redeployed payload addresses

**Review by sakulstra — APPROVED** [review]

**Comment by reem** [comment]
To limit governance fatigue and maximize the voting period during the weekday, we will push this AIP on chain on Monday Nov 20.

**Pull request merged**